ios/Podfile.lock

```diff
@@ -772,7 +772,7 @@ SPEC CHECKSUMS:
   RNLanguages: 962e562af0d34ab1958d89bcfdb64fafc37c513e
   RNPermissions: ad71dd4f767ec254f2cd57592fbee02afee75467
   RNReactNativeHapticFeedback: 2566b468cc8d0e7bb2f84b23adc0f4614594d071
-  RNReanimated: d0db0ee059c33381bca787a2193c27e52750ccaf
+  RNReanimated: 3d873b0b2e31612f09835511257d2565f61a7f77
```

Comment:
revert this file

Reply:
Yes. Thats was an accident. I saw it just late last night. 

Apologizes